Userland/Libraries/LibPDF/Fonts/PDFFont.h

```diff
@@ -27,7 +27,7 @@ class PDFFont : public RefCounted<PDFFont> {
     virtual ~PDFFont() = default;
 
     virtual void set_font_size(float font_size) = 0;
-    virtual PDFErrorOr<Gfx::FloatPoint> draw_string(Gfx::Painter&, Gfx::FloatPoint, DeprecatedString const&, Color const&, float font_size, float character_spacing, float horizontal_scaling) = 0;
+    virtual PDFErrorOr<Gfx::FloatPoint> draw_string(Gfx::Painter&, Gfx::FloatPoint, DeprecatedString const&, Color const&, float font_size, float character_spacing, float word_spacing, float horizontal_scaling) = 0;
```

Comment:
Just thinking out loud, it might make this more readable in the future to encapsulate these parameters in a struct, instead of `float, float, float, float, float` and so on. Reminds me of some of WindowServer's IPC methods with a gazillion `bool, bool, bool ...`.

Reply:
Yes, agreed. I think it's also confusing that different parameters are in different coordinate spaces. I'll try to work at this signature some more at some point.